app/views/units/_edit_form.html.erb

```diff
@@ -14,7 +14,7 @@
 
     <div class="sm:flex gap-2 items-end mb-2">
       <div>
-        <%= form.label :name, "Name hell *", class: "fancy_label"  %>
+        <%= form.label :name, "Name *", class: "fancy_label"  %>
```

Comment:
I'll do that in a separate ticket/branch since that should be done across all forms, not only sites' forms. There are some fields that don't have [required: true] in the HTML code, but are still required and need the '*' in the label. On the http://localhost:3000/programs/sites/edit_program_sites/<program_id> page, for example.